MouseAimFlight/MouseAimVesselModule.cs

```diff
@@ -152,11 +151,14 @@ void Update()
                 flightMode.NextBehavior();
                 ScreenMessages.PostScreenMessage("Flight Mode: " + flightMode.GetBehaviorName());
             }
+        }
 
+        void FixedUpdate()
+        {
             if (!mouseAimActive)
                 return;
 
-            UpdateMouseCursorForCameraRotation();
+            UpdateMouseCursorAndCameraRotation();
```

Comment:
"AND" on a method name is a code smell, because it means that the method is doing too much stuff, read [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle).
I would recomend keeping the old method, adding a method that only updates the cursor location, and using it.
Literally:
```
if(lockCamera)
    UpdateCameraRotation()
```